x-pack/solutions/security/plugins/security_solution/server/fleet_integration/handlers/create_event_filters.ts

```diff
@@ -29,24 +29,46 @@ export const createEventFilters = async (
   if (!eventFilters?.nonInteractiveSession) {
     return;
   }
+
+  // Check if the Event Filter List already exists
```

Comment:
nit: this comment is above the `try` block but does not describe the whole block correctly - could be moved, extended or, the best, removed : )